app/scripts/modules/tencentcloud/src/pipeline/stages/disableCluster/DisableCluster.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { AccountService, IStageConfigProps, NgReact, StageConfigField } from '@spinnaker/core';
+import { get } from 'lodash';
+const { AccountRegionClusterSelector } = NgReact;
+
+interface IProps extends IStageConfigProps {}
+interface IState {
+  accounts: any[];
+  regionsLoaded: boolean;
+}
+
+export class DisableCluster extends React.Component<IProps, IState> {
+  public state: IState = {
+    accounts: [],
+    regionsLoaded: false,
+  };
+
+  public componentDidMount() {
+    const { application, stage, updateStageField } = this.props;
+
+    AccountService.listAccounts('tencentcloud').then(accounts => {
+      this.setState({
+        accounts,
+      });
+    });
+
+    updateStageField({
+      cloudProvider: 'tencentcloud',
+      regions: stage.regions || [],
+    });
+
+    if (
+      stage.isNew &&
+      get(application, 'attributes.platformHealthOnlyShowOverride') &&
```

Comment:
Can likely remove some of these lodash `get`s in favor of Typescript [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining). (Same comment for other uses of `get` in other files.)